src/views/TechnicalAssistance/RequestForm/AttendeesForm/index.tsx

```diff
@@ -140,7 +140,7 @@ function AttendeesForm({
     // If no ID is present, create new attendee
     return createAttendee({
       trbRequestId,
-      euaUserId,
+      euaUserId: euaUserId || '',
```

Comment:
@samoddball I think this is where the actual EUA comes through on the component.